deployment/scripts/verify_node

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# Copyright 2020 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DEBUG="${DEBUG:-}"
+
+NODE_IP="${1:-0.0.0.0:26657}"
+
+set -eu
+set -o pipefail
+
+export PATH="$PATH:/usr/bin"
+
+echo "Checking the node 'sync_info.latest_block_height' value is not increasing..."
+
+echo "sync_info.latest_block_height Try 1"
+check1="$(dclcli status --node "$NODE_IP" | grep latest_block_height)"
+
+echo "sync_info.latest_block_height Try 2"
+sleep 6
```

Comment:
We may use a different value for empty block creation (currently it's 5 secs), so that check may need to be changed.

Reply:
added TODO comment there